_data/enterprise-landing-page-tabs-code.yml

```diff
@@ -0,0 +1,98 @@
+- id: eks
+  title: EKS cluster
+  content: |
+    <figure><figcaption>eks-cluster/main.tf</figcaption>
+      <pre>
+      <code class="language-hcl">
+        module "eks_cluster" {
+          source  = "github.com/gruntwork-io/service-catalog//eks-cluster"
+          version = "v0.51.0"
+
+          service_name  = "my-heroku-app"
```

Comment:
This param doesn't make sense for an eks-cluster... And we shouldn't mention Heroku in this experiment.

Reply:
This section has been converted to a single terraform module. This actually was on my mind, but I think with the latest features of module `depends_on`, it should be possible to have a single service module that combines `eks-cluster` and `eks-core-services` instead of having two modules like we have now.